features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/NewOverview/CustomizedStepper.jsx

```diff
@@ -401,18 +409,24 @@ export default function CustomizedStepper() {
                                                 </Grid>
                                                 <Box ml={1} mb={1}>
                                                     <Grid item>
-                                                        <Typography variant='h7'>
-                                                            <FormattedMessage
-                                                                id='Apis.Details.Overview.CustomizedStepper.Endpoint'
-                                                                defaultMessage=' Endpoint'
-                                                            />
-                                                        </Typography>
+                                                        <Link
```

Comment:
This `Link` is from `react-router-dom`

```js
import { Link } from 'react-router-dom';
```

Which ultimately create an anchor tag without any MUI styles, 

Please use the following pattern,  Use MUI Link and provide `react-router-dom` Link as its component

https://github.com/wso2/carbon-apimgt/blob/master/features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Listing/SampleAPI/SampleAPI.jsx#L335-L339

Reply:
changed as suggested

<img width="1650" alt="Screenshot 2021-04-11 at 07 05 21" src="https://user-images.githubusercontent.com/23296184/114289296-6f6a3200-9a94-11eb-9175-7fc0bd6a1ef7.png">

<img width="1631" alt="Screenshot 2021-04-11 at 07 05 37" src="https://user-images.githubusercontent.com/23296184/114289331-a2142a80-9a94-11eb-91af-a1860b5910ac.png">

Reply:
great 🥳